ansible/playbooks/common.yml

```diff
@@ -2,9 +2,10 @@
 - hosts: all
   become: true
   pre_tasks:
-    - include_vars: "../family_vars/{{ ansible_os_family | lower }}.yml"
+    - include_vars: '../family_vars/{{ ansible_os_family | lower }}.yml'
```

Comment:
Quotations, pretty sure single quotes also work, but strings which variables generally are contained within double quotes.

Reply:
Fixed IDE preference and have reverted 👍